src/components/showcase/ShowcaseHeader.js

```diff
@@ -1,5 +1,5 @@
 import lang from 'i18n-js';
```

Comment:
i believe we're not using this file anymore, maybe we should remove it

Reply:
probably not but this pr already has too many changes to keep track of so id rather remove it in a different pr so i dont break more things haha